features/page_objects/registration_details_page.rb

```diff
@@ -1,9 +1,14 @@
 class RegistrationDetailsPage < SitePrism::Page
+
   element(:site_section, :xpath, "html/body/section/main/div/div/div/div[1]/div[3]/div/div/div[1]/div/div[2]/a/span")
   element(:ngr, :xpath, ".//*[@id='site-tab']/div/table/tbody/tr[2]/td[2]")
   element(:area, :xpath, ".//*[@id='site-tab']/div/table/tbody/tr[4]/td[2]")
+  element(:registration_status, :xpath, ".//*[@id='registration']/div/table/tbody/tr[3]/td[2]/span")
   element(:deregister, "#deregister-enrollment")
-
+  element(:exemption_details_section, ".row:nth-child(3) .col-xs-11")
```

Comment:
Could we work together to move the different sections of the page into actual SitePrism sections? Appreciate in this case it won't be reused by another page, but I think it will help break up the logic and make it clear what belongs to where and what logic is relevant.